src/client/callers/agentLockAll.ts

```diff
@@ -0,0 +1,11 @@
+import { HandlerTypes, UnaryCaller } from "@matrixai/rpc";
+import NodesClosestLocalNodesGetHandler from "../../nodes/agent/handlers/NodesClosestLocalNodesGet";
+
+type CallerTypes = HandlerTypes<NodesClosestLocalNodesGetHandler>;
+
```

Comment:
Why are all the callers importing `NodesClosestLocalNodesGetHandler`?

They should import their respective handler as a type import `import type AgentLockAll from '../handlers/AgentLockAll`

Importing as a type is critical here. We're importing the handler types here so we can derive the caller types. That's why we're making this change here.